768 pbinfo/main.cpp

```diff
@@ -0,0 +1,100 @@
+#include <iostream>
+
+using namespace std;
+
+/*Se dă o matrice cu n linii şi m coloane şi elemente numere naturale. Determinați suma valorilor pare distincte din matrice.
+Dacă o valoare pară apare în matrice de mai multe ori, se va aduna o singură dată.
+
+date de intrare:
+4 6
+4 20 15 23 18 9
+1 8 23 22 14 18
+17 15 13 18 12 15
+3 18 8 20 12 5
+
+date de iesire:
+98
+*/
+
+
```

Comment:
am rezolvat mult mai intuitiv
In momentul in care faci citirea poti aduna la suma daca numarul corespunde cu cerintele problemei
Cerinte:
1. paritate 
- `if (nr % 2 == 0)`
2. distinctie
- cand gasesti un numar par care nu apare in arr[], il adaugi la suma si il pui in arr[] ca sa nu-l mai adaugi daca mai apare o data
- observatie, ca sa ti minte lungimea arr[] setezi un counter pe care-l incrementezi de fiecare data cand salvezi un nr in arr[]